fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/typecoercion/ExpectsInputTypes.java

```diff
@@ -35,4 +35,8 @@
 public interface ExpectsInputTypes {
 
     List<AbstractDataType> expectedInputTypes();
+
+    default void checkInputTypes() {
+
+    }
```

Comment:
why we need a blank check?

Reply:
for extends to check input types in the future